tests/framework/clients/pool.go

```diff
@@ -111,7 +111,10 @@ func (p *PoolOperation) DeletePool(blockClient *BlockOperation, namespace, poolN
 	for _, blockImage := range blockImagesList {
 		if poolName == blockImage.PoolName {
 			logger.Infof("force deleting block image %q in pool %q", blockImage, poolName)
-			blockClient.DeleteBlockImage(blockImage, namespace)
+			err := blockClient.DeleteBlockImage(blockImage, namespace)
+			if err != nil {
+				logger.Infof("failed deleting image %q from %q. %v", blockImage, poolName, err)
```

Comment:
have you seen block images failing to get deleted in the tests?

Reply:
The thing is one error is expected when the controller fails to remove the pool because it has images, so my assumption is that the force deletion failed for some reason in the CI.